patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -834,6 +834,20 @@
 +    }
 +
 +    /**
++     * Allows a block to override the standard Entity.createRunningParticles particles.
++     * 
++     * @param state State at the specific world/pos
++     * @param worldObj The current world
++     * @param pos Position of the block that the entity is running on.
++     * @param entity The entity that is running on the block.
++     * @return True to prevent vanilla running particles form spawning.
++     */
++    public boolean addRunningEffects(IBlockState state, World worldObj, BlockPos pos, Entity entity)
```

Comment:
I was going to but notice the one above and bellow used worldObj and with them all being similar methods I left it. Most of the others have been cleaned up though so I'll change this as well.